docs-ref-services/storage.md

```diff
@@ -1,62 +1,41 @@
 ---
-title: Azure Storage modules for Node.js
-description: Reference for Azure Storage modules for Node.js
+title: Azure Storage modules for JavaScript
+description: Reference for Azure Storage modules for JavaScript
 author: craigshoemaker
 ms.author: cshoe
 manager: routlaw
-ms.date: 07/18/2017
+ms.date: 01/16/2019
 ms.topic: article
 ms.prod: azure
 ms.technology: azure
-ms.devlang: nodejs
+ms.devlang: javascript
 ms.service: storage
 ---
 
-# Azure Storage modules for Node.js
+# Azure Storage libraries for JavaScript
 
-Use the Azure Storage client module to:
+[Azure Storage](https://docs.microsoft.com/azure/storage/) is a Microsoft-managed service providing cloud storage that is highly available, secure, durable, scalable, and redundant. The following libraries in JavaScript make it easy to consume Azure Storage service.
```

Comment:
Why the hyphenated `Microsoft-manager`?

Reply:
Not me. Copy/pasted from the `azure-sdk-for-js`/`storage` readme.